fuzzing/run_fuzzers.py

```
@@ -0,0 +1,487 @@
+        """Validate environment for fuzzing"""
+        print("🔍 Validating environment...")
+
+        # Check available disk space (need at least 1GB)
+        try:
+            stat = os.statvfs(self.fuzz_dir)
+            free_space_gb = (stat.f_bavail * stat.f_frsize) / (1024**3)
+            if free_space_gb < 1.0:
+                print(f"❌ Insufficient disk space: {free_space_gb:.1f}GB available, need at least 1GB")
+                return False
+            print(f"✅ Disk space: {free_space_gb:.1f}GB available")
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)